test/e2e/framework/rc_util.go

```diff
@@ -156,7 +115,7 @@ func DeleteRCAndWaitForGC(c clientset.Interface, ns, name string) error {
 	return DeleteResourceAndWaitForGC(c, api.Kind("ReplicationController"), ns, name)
 }
 
-// ScaleRC scales Replication Controller to be desired size.
+// ScaleRC scales replication controller to be desired size.
```

Comment:
Replication Controller should be upper case as in the official docs:
https://kubernetes.io/docs/tasks/run-application/rolling-update-replication-controller/